index.html

```diff
@@ -0,0 +1,997 @@
+            <div class="badge">
+              <img src="https://cdn.discordapp.com/avatars/778699572787675136/bfd4785c0eb8a2be4eae7cc2fec27d34.png?size=256&quot" class="avatar-sm"/>
+              <span class="badge-avatar-text-sm">41</span>
+            </div>
+            <div class="card-badge-body">
+              <div class="card-badge-head-sm">Hello</div>
+              <div class="card-badge-text-sm">Some msg </div>
+            </div>           
+          </div>
+          <div class="code">
+            <pre>
+              <code class="language-html">
+                &#x3C;div class=&#x22;card card-badge card-body&#x22; style=&#x22;width: 15rem;&#x22;&#x3E;
+                &#x3C;div class=&#x22;badge&#x22;&#x3E;
+                  &#x3C;img src=... class=&#x22;avatar-lg&#x22;/&#x3E;
+                  &#x3C;span class=&#x22;badge-avatar-text-lg&#x22;&#x3E;41&#x3C;/span&#x3E;
+                &#x3C;/div&#x3E;
+                &#x3C;div class=&#x22;card-badge-body&#x22;&#x3E;
+                  &#x3C;div class=&#x22;card-badge-head-lg&#x22;&#x3E;Hello&#x3C;/div&#x3E;
+                  &#x3C;div class=&#x22;card-badge-text-lg&#x22;&#x3E;Some message&#x3C;/div&#x3E;
+                &#x3C;/div&#x3E;
+              &#x3C;/div&#x3E;
+  
+              &#x3C;div class=&#x22;card card-badge card-body&#x22; style=&#x22;width: 12rem;&#x22;&#x3E;
+                &#x3C;div class=&#x22;badge&#x22;&#x3E;
+                  &#x3C;img src=... class=&#x22;avatar-md&#x22;/&#x3E;
+                  &#x3C;span class=&#x22;badge-avatar-text-md&#x22;&#x3E;41&#x3C;/span&#x3E;
+                &#x3C;/div&#x3E;
+                &#x3C;div class=&#x22;card-badge-body&#x22;&#x3E;
+                  &#x3C;div class=&#x22;card-badge-head-md&#x22;&#x3E;Hello&#x3C;/div&#x3E;
+                  &#x3C;div class=&#x22;card-badge-text-md&#x22;&#x3E;Some msg &#x3C;/div&#x3E;
+                &#x3C;/div&#x3E;
+              &#x3C;/div&#x3E;
+  
+              &#x3C;div class=&#x22;card card-badge card-body&#x22; style=&#x22;width: 8rem;&#x22;&#x3E;
+                &#x3C;div class=&#x22;badge&#x22;&#x3E;
+                  &#x3C;img src=... class=&#x22;avatar-sm&#x22;/&#x3E;
+                  &#x3C;span class=&#x22;badge-avatar-text-sm&#x22;&#x3E;41&#x3C;/span&#x3E;
+                &#x3C;/div&#x3E;
+                &#x3C;div class=&#x22;card-badge-body&#x22;&#x3E;
+                  &#x3C;div class=&#x22;card-badge-head-sm&#x22;&#x3E;Hello&#x3C;/div&#x3E;
+                  &#x3C;div class=&#x22;card-badge-text-sm&#x22;&#x3E;Some msg &#x3C;/div&#x3E;
+                &#x3C;/div&#x3E;
+              &#x3C;/div&#x3E;
+              </code>
+            </pre>
+          </div>
+
+          <h4>Cards with text overlay</h4>
+          <div class="card" style="width: 15rem;">
+            <img src="https://cdn.discordapp.com/avatars/778699572787675136/bfd4785c0eb8a2be4eae7cc2fec27d34.png?size=256&quot" class="card-image">
+            <div class="card-text-overlay">
```

Comment:
Most overlay are usually rgba values of black. This works as well though, just nit picking.